http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java

```diff
@@ -60,6 +60,7 @@
     private String kerberosPrincipal;
     private String kerberosRemoteServiceName;
     private int selectorThreads = 2;
+    private boolean syncSocketAddressResolutionEnabled;
```

Comment:
Nit, but I think this sound better named `useSyncSocketAddressResolution`. Compare
```
http-client.sync-socket-address-resolution-enabled=true
http-client.use-sync-socket-address-resolution=true
```
Trying to break down why, "enabled" seems more appropriate for something like HTTP/2 or authentication that we enable to allow it to be used, whereas "use" is better for "always use this thing".